src/pages/public_registration/action_accordions/FindRegistration.tsx

```diff
@@ -0,0 +1,122 @@
+import { Registration } from '../../../types/registration.types';
+import { Autocomplete, Box, IconButton, TextField, Typography } from '@mui/material';
+import { dataTestId } from '../../../utils/dataTestIds';
+import SearchIcon from '@mui/icons-material/Search';
+import { useQuery } from '@tanstack/react-query';
+import { fetchResults } from '../../../api/searchApi';
+import { useState } from 'react';
+import { useDebounce } from '../../../utils/hooks/useDebounce';
+import { useTranslation } from 'react-i18next';
+import { ErrorBoundary } from '../../../components/ErrorBoundary';
+import { SearchListItem } from '../../../components/styled/Wrappers';
+import { RegistrationListItemContent } from '../../../components/RegistrationList';
+
+function isTitle(query: string) {
+  return !isDoi(query) && !isHandle(query);
+}
+
+function isDoi(query: string) {
+  return query.includes('https://doi.org/');
+}
+
+function isHandle(query: string) {
+  return query.includes('https://hdl.handle.net/');
+}
+
+interface FindRegistrationProps {
+  setSelectedRegistration: (registration: Registration | null) => void;
+  selectedRegistration: Registration | null;
+  filteredRegistrationIdentifier: string;
+}
+export const FindRegistration = ({
+  setSelectedRegistration,
+  selectedRegistration,
+  filteredRegistrationIdentifier,
+}: FindRegistrationProps) => {
+  const { t } = useTranslation();
+  const [searchBeforeDebounce, setSearchBeforeDebounce] = useState('');
+  const debouncedSearch = useDebounce(searchBeforeDebounce);
+
+  const fetchQuery = {
+    title: isTitle(debouncedSearch) ? debouncedSearch : null,
+    doi: isDoi(debouncedSearch) ? debouncedSearch : null,
+    query: isHandle(debouncedSearch) ? debouncedSearch : null,
```

Comment:
Liker at det er delt opp og søker på det som er mest spesifikt! 
Men lurer på om vi skulle gått rett på `query` om man ikke har oppgitt en doi? Da kan jeg søke opp alle mine resultat også, ved å søke på "kjetil møkkelgjerd", og vil ikke miste noen andre treff
```suggestion
    doi: isDoi(debouncedSearch) ? debouncedSearch : null,
    query: !isDoi(debouncedSearch) ? debouncedSearch : null,
```